refinery/file_store/test_utils.py

```diff
@@ -4,22 +4,66 @@
 from django.conf import settings
 from django.test import SimpleTestCase
 
-from .utils import SymlinkedFileSystemStorage
+import mock
+
+from .utils import S3MediaStorage, SymlinkedFileSystemStorage
+
+
+class S3MediaStorageTest(SimpleTestCase):
+
+    def setUp(self):
+        self.storage = S3MediaStorage(bucket_name='test-bucket')
+
+    @mock.patch('file_store.utils.S3MediaStorage.exists')
+    def test_file_name_format(self, mock_exists):
+        mock_exists.return_value = False
+        name = self.storage.get_available_name('test.fastq')
+        self.assertRegexpMatches(name, '[\w]{7}/test\.fastq')
```

Comment:
What does the 7 correlate to here?

I'm guessing: `/xx/xx/`? Maybe a more explicit regex would be clearer?

Reply:
That 7 corresponds to the one in [S3MediaStorage.get_available_name()](https://github.com/refinery-platform/refinery-platform/pull/2737/files#diff-e861c3dd97417068aa544e06b9ba89fdR22) which is what's being tested here.

Forward slash is not a special character: https://docs.python.org/2/library/re.html#regular-expression-syntax